templates/navigation-bar.twig

```diff
@@ -19,6 +19,7 @@
 			</span>
 		</button>
 		<ul id="navbar-dropdown" class="navbar-dropdown list-unstyled">
```

Comment:
Not sure why do you need an id here. Correct me if I'm wrong, but I don't see any use with it.

Reply:
Not my code :grin:  but it's used by the current country selector script as a selector (although probably not needed even for that) - I will remove this in the new navbar design implementation